bitbots_bringup/scripts/check_robot.py

```diff
@@ -92,18 +92,17 @@ def check_pressure(msg, min, max, foot_name):
 def pwm_callback(msg:JointState):
     global got_pwm
     for effort in msg.effort:
-        if effort = 100:
+        if effort == 100:
             print_warn("Servo reported max PWM value. It is working at its limit!\n")
 
-#todo set correctly
-ACCLEL_LIMIT = 0
-ANGULAR_VEL_LIMIT = 0
+ACCEL_LIMIT = 35
+ANGULAR_VEL_LIMIT = 10
 def imu_callback(msg: Imu):
     for accel in msg.linear_acceleration:
-        if accel > ACCEL_LIMIT:
+        if accel < -ACCEL_LIMIT or accel > ACCEL_LIMIT:
```

Comment:
```suggestion
        if abs(accel) > ACCEL_LIMIT:
```
would be more readable (I think)